scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -263,6 +285,40 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
                 }
             }
 
+            /**
+             * Enable Disable the Send and Publish button.
+             * Send And Publish is enabled using `superdesk.config.js`.
+             */
+            scope.showSendAndPublish = () => !config.ui || angular.isUndefined(config.ui.sendAndPublish) ||
+                                             config.ui.sendAndPublish;
```

Comment:
Let's indent 4 spaces here if you don't mind. We are trying to promote this throughout the project. I promise I will update the linters soon :-)